core/services/commander/main.py

```diff
@@ -115,7 +115,7 @@ async def shutdown(response: Response, shutdown_type: ShutdownType, i_know_what_
             output = run_command(f"(sleep {hold_time_seconds}; sudo shutdown --reboot -h now)&")
             logger.debug(f"reboot: {output}")
         elif shutdown_type == ShutdownType.POWEROFF:
-            output = run_command(f"(sleep {hold_time_seconds}; sudo shutdown --poweroff -h now)&")
+            output = run_command(f"(sleep {hold_time_seconds}; sudo reboot)&")
```

Comment:
This is replacing power off and not rebooting, is this correct ?

Reply:
lol it is not